src/plugins/core/data_validation.ts

```diff
@@ -122,6 +122,24 @@ export class DataValidationPlugin
         this.addDataValidationRule(cmd.sheetId, { ...cmd.rule, ranges });
         break;
       }
+      case "DELETE_CONTENT": {
```

Comment:
what about an UPDATE_CELL with `content: ""` ?

Reply:
It's a strange case in fact. For example, selecting the cell and emptying the composer should do nothing to the checkbox, but copying an empty cell over the "checkboxed" cell should clear it, so we should have a way to differentiate these things

Reply:
ok, fair if it's done on purpose :+1: 
Is it tested ?